controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -760,20 +758,7 @@ protected Scope getScope(final String scopeName) {
 
         return result;
     }
-
-    private CompletableFuture<SimpleEntry<Long, Long>> findNumSplitsMerges(String scopeName, String streamName, Executor executor) {
```

Comment:
Where is this moving to? A browser search is not giving me anything...

Reply:
I have removed this.